src/Adapter/ElFinder/PhpcrDriver.php

```diff
@@ -31,7 +30,7 @@
 /**
  * @author Sjoerd Peters <sjoerd.peters@gmail.com>
  */
-class PhpcrDriver extends ElFinderVolumeDriver
+class PhpcrDriver extends \elFinderVolumeDriver
```

Comment:
this will break with older versions of elfinder bundle, however. can you bump the version in composer to the one that uses the new elfinder library?

Reply:
I added helios-ag/fm-elfinder-bundle < 8.0 to the conflict section.